README.md

```diff
@@ -68,4 +68,15 @@ Search for “sumologic-mongodb-atlas” and select the app as shown below:
         * Private API Key: Paste the Private Key from step 1.
         * Public API Key: Paste the Public Key from step 1.
     * Click Deploy.
-
```

Comment:
Make this as a separate section and add a title above these steps.
#Whitelisting Lambda's IP Address

Reply:
okay